test/pycheck/conftest.py

```diff
@@ -62,10 +63,13 @@ def conn(pg):
 @pytest.fixture
 def md_cur(pg, ddb, request):
     """A cursor to a MotherDuck enabled pg_duckdb"""
-    _ = ddb  # silence warning, we only need ddb the ddb
+    _ = ddb  # silence warning, we only need ddb
     test_db = request.node.name.removeprefix("test_")
-    pg.configure("duckdb.motherduck_enabled = true")
-    pg.restart()
+
+    if not pg.md_setup:
```

Comment:
This `md_setup` boolean isn't needed anymore afaict, because `enable_motherduck()` is a no-op if it's already enabled.